src/bootloader-shared.ts

```diff
@@ -26,24 +26,26 @@
  */
 export const qrlResolver = (
   doc: Document,
-  eventUrl: string | null | undefined,
-  linkElm?: HTMLLinkElement,
-  href?: string,
-  url?: URL
+  element: Element | null,
+  eventUrl?: string | null,
+  _url?: string,
```

Comment:
Not important, but why the `_` here?

Reply:
because it is cut and paste from a bootloader where they are parameters to save bytes and I wanted to make it clear that they are not to be used.